clean-abap/CleanABAP.md

```diff
@@ -866,6 +867,26 @@ ENDDO.
 
 > Read more in _Chapter 17: Smells and Heuristics: G27: Structure over Convention_ of [Robert C. Martin's _Clean Code_].
 
+### Constants also need descriptive names
+
+> [Clean ABAP](#clean-abap) > [Content](#content) > [Constants](#constants) > [This section](#constants-also-need-descriptive-names)
+
+There is a historic tendency in ABAP to wrap every literal in constants with names 
+that merely repeat their content or even their type:
+```ABAP
+" anti-pattern 
+CONSTANTS: 
+  c_01 TYPE spart VALUE '01',
+  c_mmsta TYPE mmsta VALUE '90'.
+```
+There is little benefit to either variant. It is not informative for the reader, and if 
+the value ever needs to change then a constant that parrots its value must also be renamed. 
+
+If a constant is declared in code then it should describe its meaning not its content.
+```ABAP
+CONSTANTS c_status_inactive TYPE mmsta VALUE '90'.
```

Comment:
The "good" example should not contain any prefix, see rule "Avoid encodings, esp. Hungarian notation and prefixes"